app/views/hyrax/my/_collection_action_menu.html.erb

```diff
@@ -45,9 +44,11 @@
       <% end %>
     </li>
 
+    <%# OVERRIDE begin %>
     <% if Hyrax::CollectionType.any_nestable? %>
+    <%# OVERRIDE end %>
```

Comment:
🟡  How would this if line be the beginning and end of an override?

Reply:
You're right there are a few smaller things other than this line, i'll remove the comment